packages/node_modules/@ciscospark/widget-roster/src/container.js

```diff
@@ -4,39 +4,56 @@ import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import classNames from 'classnames';
 import {autobind} from 'core-decorators';
+import debounce from 'lodash/debounce';
```

Comment:
For treeshaking and the lodash babel filter I think this needs to be `import {debounce} from 'lodash';`